addons/mail/static/tests/qunit_suite_tests/components/chatter_topbar_tests.js

```diff
@@ -51,9 +51,9 @@ QUnit.test('base rendering', async function (assert) {
         "attachments button should not have a loader"
     );
     assert.strictEqual(
-        document.querySelectorAll(`.o_ChatterTopbar_buttonAttachmentsCount`).length,
+        document.querySelectorAll(`.o_ChatterTopbar_buttonAddAttachments_text`).length,
```

Comment:
`_` is only used to separate component from part, should not appear twtice in part name